src/main/resources/templates/close.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html xmlns:th = "http://www.thymeleaf.org">
+<html lang="en" xmlns:th = "http://www.thymeleaf.org">
```

Comment:
Another small fix to define the language within the HTML tag. Resolves validation errors.